generator/src/index.rs

```diff
@@ -148,3 +162,42 @@ impl Export {
         Ok(())
     }
 }
+
+/// Check membership of one or moreh3 indexes in all h3idz files in a given
+/// folder
+#[derive(Debug, clap::Args)]
+pub struct Find {
+    input: path::PathBuf,
+    cells: Vec<h3ron::H3Cell>,
```

Comment:
I see what you're doing here, and why you have to choose between a `PathBuf, Vec<H3Cell>` or `Vec<PathBuf>, H3Cell` due to positional arguments. But my preference would be to have a single Cell and and `Vec<PathBuf>`. This way, the user can call `find <SOMEHEX> path/to/indicies/*` or even `find <SOMEHEX> path/to/indicies/*.h3idz`.

Note that #29 uses `Vec<PathBuf>`, so we should use the same convention regardless which we choose.

Reply:
I opted for a folder argument which would be searched for all  h3idz files to look for multiple cells in. This seems to be how we've been wanting to use it to find multiple locations at once